README.md

```diff
@@ -48,9 +48,30 @@ To do this, set:
 
 For example:
 
-    vault write secret/foo value=bar other=baz
+    vault write secret/foo value=bar
+
+The hiera lookup for `foo` will return just "bar" as a string, but only if it is the only field.
+If `foo` contains more fields, a Hash will be returned, just like with the default behaviour.
+
+### json parsing of single values - optional
```

Comment:
Since this option depends on `default_field`, can this be a sub-section instead, i.e. `####` rather than `###`.

Also, JSON in uppercase.